.github/workflows/issue-welcome.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  issues:
+    types:
+      - opened
+
+# https://github.com/marketplace/actions/create-or-update-comment
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/create-or-update-comment@v4
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            👋 Thanks for opening this issue!
+
+            Get help or engage by:
```

Comment:
Woud you like the command of `/good-first-issue` for label of `good first issue`?

Reply:
It's better to let maintainer to assign the tag instead of the issue creator.